tests/pipeline-all-amps/run.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+IMAGE_TAG=7.0.0-M2
+
+sed -e "s@\${image.tag}@$IMAGE_TAG@" Dockerfile > target/Dockerfile
+cp ../docker-entrypoint.sh target/
+
+docker rmi -f $DOCKER_IMAGE_REPO
+docker build -f target/Dockerfile -t $DOCKER_IMAGE_REPO .
+echo http://localhost:8080$BASE_PATH
+# --user 1000:1000 \
+docker run --rm -it \
+  --publish 8080:8080 $DOCKER_IMAGE_REPO "$@"
```

Comment:
I'm not sure what the expected use case for this script is, but it seems to lose some value by picking 8080 for both repo and share.

Reply:
local testing with standard tool but we can do such change if we want to use them as part of some env test